src/Algebra/Graph.hs

```diff
@@ -1041,3 +1054,48 @@ sparsify graph = res
         m <- get
         put (m + 1)
         overlay <$> s `x` m <*> m `y` t
+
+buildG :: forall a. F.Fold a -> Graph a
+buildG g = F.foldg Empty Vertex Overlay Connect g
+{-# INLINE [1] buildG #-}
+
+-- | FB functions
```

Comment:
Please add some comments for all these FB functions. Don't hesitate to write long comments: you can even add a part of your blog post here!

Reply:
Yes, indeed "FB functions" are maybe not enough ^^

Reply:
I added a modified version of the comment in `GHC.Base`: http://hackage.haskell.org/package/base-4.11.0.0/docs/src/GHC.Base.html#mapFB